grakn-test/test-unit/src/test/java/ai/grakn/test/engine/lock/LockTestIT.java

```diff
@@ -180,7 +180,7 @@ public void whenGettingLockWithDotInPath_LockIsAcquired(Locks locks){
 
     @Theory
     public void whenGettingLockWithDoubleDotInPath_LockIsAcquired(Locks locks){
-        String lockPath = "/..";
+        String lockPath = "/src/main";
```

Comment:
Are these locks actually filesystem paths? I don't know enough about how this works to say.